pxr/usd/sdf/wrapPredicateExpression.cpp

```diff
@@ -96,9 +101,7 @@ void wrapPredicateExpression()
         .def(self == self)
         .def(self != self)
 
-        .def("GetParseError",
```

Comment:
<details>
<summary>Here is the error spew that prompted this change. In this case I was using Boost 1.82.0, but I got the same result using the current Windows/VS2022 default of 1.78.0.</summary>

```
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): error C2672: 'get_signature': no matching overloaded function found [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : could be 'boost::mpl::vector17<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector17<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector17<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector17<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector17<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector17<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector17<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector17<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector17<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector17<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector17<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector17<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector17<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector17<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector17<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector17<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector17<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector17<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector16<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14,T15> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector16<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14,T15> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector16<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector16<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector16<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector16<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector16<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector16<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector16<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector16<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector16<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector16<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector16<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector16<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector16<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector16<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector16<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector16<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector16<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector16<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector15<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector15<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13,T14> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector15<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector15<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector15<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector15<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector15<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector15<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector15<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector15<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector15<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector15<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector15<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector15<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector15<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector15<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector15<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector15<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector15<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector15<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector14<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector14<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12,T13> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector14<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector14<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector14<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector14<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector14<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector14<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector14<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector14<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector14<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector14<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector14<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector14<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector14<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector14<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector14<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector14<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector14<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector14<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector13<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector13<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11,T12> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector13<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector13<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector13<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector13<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector13<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector13<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector13<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector13<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector13<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector13<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector13<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector13<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector13<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector13<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector13<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector13<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector13<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector13<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector12<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector12<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10,T11> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector12<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector12<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector12<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector12<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector12<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector12<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector12<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector12<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector12<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector12<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector12<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector12<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector12<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector12<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector12<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector12<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector12<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector12<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector11<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector11<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9,T10> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8,T9)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector11<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector11<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector11<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector11<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector11<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector11<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector11<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector11<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector11<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector11<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector11<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector11<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector11<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector11<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector11<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector11<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector11<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector11<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7,T8))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector10<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector10<T0,T1,T2,T3,T4,T5,T6,T7,T8,T9> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7,T8)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector10<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector10<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector10<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector10<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector10<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector10<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector10<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector10<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector10<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector10<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector10<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector10<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector10<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector10<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector10<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector10<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector10<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector10<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6,T7))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector9<T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector9<T0,T1,T2,T3,T4,T5,T6,T7,T8> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6,T7)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector9<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector9<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector9<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector9<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector9<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector9<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector9<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector9<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector9<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector9<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector9<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector9<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector9<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector9<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector9<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector9<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector9<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector9<RT,ClassT&,T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5,T6))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector8<T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector8<T0,T1,T2,T3,T4,T5,T6,T7> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1,T2,T3,T4,T5,T6)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector8<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector8<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector8<RT,ClassT&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector8<RT,ClassT&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector8<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector8<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector8<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector8<RT,ClassT&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector8<RT,ClassT&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector8<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector8<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector8<RT,ClassT&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector8<RT,ClassT&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector8<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector8<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector8<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector8<RT,ClassT&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector8<RT,ClassT&,T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4,T5))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector7<T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector7<T0,T1,T2,T3,T4,T5,T6> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4,T5),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1,T2,T3,T4,T5)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector7<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector7<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector7<RT,ClassT&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector7<RT,ClassT&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector7<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector7<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector7<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector7<RT,ClassT&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector7<RT,ClassT&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector7<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector7<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector7<RT,ClassT&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector7<RT,ClassT&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector7<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector7<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector7<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector7<RT,ClassT&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector7<RT,ClassT&,T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3,T4))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector6<T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector6<T0,T1,T2,T3,T4,T5> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3,T4),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1,T2,T3,T4)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector6<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector6<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector6<RT,ClassT&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector6<RT,ClassT&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector6<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector6<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector6<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector6<RT,ClassT&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector6<RT,ClassT&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector6<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector6<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector6<RT,ClassT&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector6<RT,ClassT&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector6<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector6<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector6<RT,most_derived<Target,ClassT>::type&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2,T3)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector6<RT,ClassT&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector6<RT,ClassT&,T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2,T3))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector5<T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector5<T0,T1,T2,T3,T4> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2,T3),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1,T2,T3)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector5<RT,most_derived<Target,ClassT>::type&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector5<RT,most_derived<Target,ClassT>::type&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector5<RT,ClassT&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector5<RT,ClassT&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector5<RT,most_derived<Target,ClassT>::type&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector5<RT,most_derived<Target,ClassT>::type&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector5<RT,most_derived<Target,ClassT>::type&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector5<RT,ClassT&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector5<RT,ClassT&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector5<RT,most_derived<Target,ClassT>::type&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector5<RT,most_derived<Target,ClassT>::type&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector5<RT,ClassT&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector5<RT,ClassT&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector5<RT,most_derived<Target,ClassT>::type&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector5<RT,most_derived<Target,ClassT>::type&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector5<RT,most_derived<Target,ClassT>::type&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1,T2)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector5<RT,ClassT&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector5<RT,ClassT&,T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1,T2))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector4<T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector4<T0,T1,T2,T3> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1,T2),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1,T2)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector4<RT,most_derived<Target,ClassT>::type&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector4<RT,most_derived<Target,ClassT>::type&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector4<RT,ClassT&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector4<RT,ClassT&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector4<RT,most_derived<Target,ClassT>::type&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector4<RT,most_derived<Target,ClassT>::type&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector4<RT,most_derived<Target,ClassT>::type&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector4<RT,ClassT&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector4<RT,ClassT&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector4<RT,most_derived<Target,ClassT>::type&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector4<RT,most_derived<Target,ClassT>::type&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector4<RT,ClassT&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector4<RT,ClassT&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector4<RT,most_derived<Target,ClassT>::type&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector4<RT,most_derived<Target,ClassT>::type&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector4<RT,most_derived<Target,ClassT>::type&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0,T1)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector4<RT,ClassT&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector4<RT,ClassT&,T0,T1> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0,T1))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector3<T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector3<T0,T1,T2> boost::python::detail::get_signature(RT (__cdecl *)(T0,T1),void *)': could not deduce template argument for 'RT (__cdecl *)(T0,T1)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector3<RT,most_derived<Target,ClassT>::type&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector3<RT,most_derived<Target,ClassT>::type&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector3<RT,ClassT&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector3<RT,ClassT&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector3<RT,most_derived<Target,ClassT>::type&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector3<RT,most_derived<Target,ClassT>::type&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector3<RT,most_derived<Target,ClassT>::type&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector3<RT,ClassT&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector3<RT,ClassT&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector3<RT,most_derived<Target,ClassT>::type&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector3<RT,most_derived<Target,ClassT>::type&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector3<RT,ClassT&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector3<RT,ClassT&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector3<RT,most_derived<Target,ClassT>::type&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector3<RT,most_derived<Target,ClassT>::type&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector3<RT,most_derived<Target,ClassT>::type&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(T0)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector3<RT,ClassT&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector3<RT,ClassT&,T0> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(T0))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector2<T0,T1> boost::python::detail::get_signature(RT (__cdecl *)(T0),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector2<T0,T1> boost::python::detail::get_signature(RT (__cdecl *)(T0),void *)': could not deduce template argument for 'RT (__cdecl *)(T0)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector2<RT,most_derived<Target,ClassT>::type&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void) volatile const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector2<RT,most_derived<Target,ClassT>::type&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void) volatile const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(void) volatile const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector2<RT,ClassT&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void) volatile const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector2<RT,ClassT&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void) volatile const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector2<RT,most_derived<Target,ClassT>::type&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void) volatile,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector2<RT,most_derived<Target,ClassT>::type&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void) volatile,Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector2<RT,most_derived<Target,ClassT>::type&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void) volatile,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(void) volatile' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector2<RT,ClassT&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void) volatile)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector2<RT,ClassT&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void) volatile)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector2<RT,most_derived<Target,ClassT>::type&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void) const,Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector2<RT,most_derived<Target,ClassT>::type&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void) const,Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(void) const' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector2<RT,ClassT&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void) const)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector2<RT,ClassT&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void) const)': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(236,1): message : or       'boost::mpl::vector2<RT,most_derived<Target,ClassT>::type&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void),Target *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector2<RT,most_derived<Target,ClassT>::type&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void),Target *)': template parameter 'ClassT' is ambiguous [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : could be 'const pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : or       'pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector2<RT,most_derived<Target,ClassT>::type&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void),Target *)': could not deduce template argument for 'RT (__cdecl ClassT::* )(void)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(218,1): message : or       'boost::mpl::vector2<RT,ClassT&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void))' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector2<RT,ClassT&> boost::python::detail::get_signature(RT (__cdecl ClassT::* )(void))': expects 1 arguments - 2 provided [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/signature.hpp(194,1): message : or       'boost::mpl::vector1<T0> boost::python::detail::get_signature(RT (__cdecl *)(void),void *)' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(497,25): message : 'boost::mpl::vector1<T0> boost::python::detail::get_signature(RT (__cdecl *)(void),void *)': could not deduce template argument for 'RT (__cdecl *)(void)' from 'Fn' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(562,13): message : see reference to function template instantiation 'void boost::python::class_<PredExpr,boost::python::detail::not_specified,boost::python::detail::not_specified,boost::python::detail::not_specified>::def_impl<pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression,Fn,boost::python::detail::def_helper<A1,boost::python::detail::not_specified,boost::python::detail::not_specified,boost::python::detail::not_specified>>(T *,const char *,Fn,const Helper &,...)' being compiled [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &,
              A1=boost::python::return_value_policy<boost::python::return_by_value,boost::python::default_call_policies>,
              T=pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression,
              Helper=boost::python::detail::def_helper<boost::python::return_value_policy<boost::python::return_by_value,boost::python::default_call_policies>,boost::python::detail::not_specified,boost::python::detail::not_specified,boost::python::detail::not_specified>
          ]
D:\OpenUSD_test_inst\include\boost-1_82\boost/python/class.hpp(240,13): message : see reference to function template instantiation 'void boost::python::class_<PredExpr,boost::python::detail::not_specified,boost::python::detail::not_specified,boost::python::detail::not_specified>::def_maybe_overloads<A1,A2>(const char *,Fn,const boost::python::return_value_policy<boost::python::return_by_value,boost::python::default_call_policies> &,...)' being compiled [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              A1=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &,
              A2=boost::python::return_value_policy<boost::python::return_by_value,boost::python::default_call_policies>,
              Fn=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &
          ]
C:\Users\MattJohnson\Developer\OpenUSD\pxr\usd\sdf\wrapPredicateExpression.cpp(99,9): message : see reference to function template instantiation 'boost::python::class_<PredExpr,boost::python::detail::not_specified,boost::python::detail::not_specified,boost::python::detail::not_specified> &boost::python::class_<PredExpr,boost::python::detail::not_specified,boost::python::detail::not_specified,boost::python::detail::not_specified>::def<const std::string&(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &,boost::python::return_value_policy<boost::python::return_by_value,boost::python::default_call_policies>>(const char *,A1,const A2 &)' being compiled [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
          with
          [
              A1=const std::string &(__cdecl pxrInternal_v0_23__pxrReserved__::SdfPredicateExpression::* )(void) const &,
              A2=boost::python::return_value_policy<boost::python::return_by_value,boost::python::default_call_policies>
          ]
C:\Users\MattJohnson\Developer\OpenUSD\pxr\usd\sdf\wrapPredicateExpression.cpp(59,13): message : see the first reference to 'boost::python::class_<PredExpr,boost::python::detail::not_specified,boost::python::detail::not_specified,boost::python::detail::not_specified>::def' in 'wrapPredicateExpression' [D:\OpenUSD_test_inst\build\OpenUSD\pxr\usd\sdf\_sdf.vcxproj]
```
</details>

Reply:
I wasn't able to reproduce these errors in a standard build with C++20 enabled and using the new internal pxr_boost::python library. I *was* able to reproduce this if I switched the build back to using boost::python though.

So this change is only needed if you're modifying your OpenUSD build to use C++20 *and* using the backwards-compatibility option to use boost::python, which is slated for removal in the near future. Given that, I'm kind of inclined to revert the change here. What do you think?

Reply:
That sounds great to me! I dropped this commit in what I just pushed.